frontend/src/components/licences/LicenceDetails.vue

```diff
@@ -0,0 +1,119 @@
+            </v-row>
+          </v-col>
+          <v-col cols="12" md="6" lg="5" xl="6">
+            <v-row no-gutters class="mr-2 my-2">
+              <v-col cols="12" sm="6" xl="4" class="pr-2">
+                <AppLabel>Weeks in Operation:</AppLabel>
+              </v-col>
+              <v-col cols="12" sm="6" xl="8">{{ licence?.weeksInOperation }}</v-col>
+            </v-row>
+          </v-col>
+          <v-col cols="12" md="6" lg="3">
+            <v-row no-gutters class="mr-2 my-2">
+              <v-col cols="12" sm="6" xl="5" class="pr-2">
+                <AppLabel>Care Type:</AppLabel>
+              </v-col>
+              <v-col cols="12" sm="6" xl="7">{{ licence?.careType === 1 ? 'Full-Time' : 'Part-Time' }}</v-col>
+            </v-row>
+          </v-col>
+          <v-col cols="12" md="6" lg="4" xl="3">
+            <v-row no-gutters class="mr-2 my-2">
+              <v-col cols="12" sm="6" class="pr-2">
+                <AppLabel>Operational Spaces:</AppLabel>
+              </v-col>
+              <v-col cols="12" sm="6">{{ licence?.operationalSpaces }}</v-col>
+            </v-row>
+          </v-col>
+          <v-col cols="12" md="6" lg="5" xl="6">
+            <v-row no-gutters class="mr-2 my-2">
+              <v-col cols="12" sm="6" xl="4" class="pr-2">
+                <AppLabel>Days of Week:</AppLabel>
+              </v-col>
+              <v-col cols="12" sm="6" xl="8">{{ convertDaysToString(licence?.weekDays) }}</v-col>
+            </v-row>
+          </v-col>
+          <v-col cols="12" md="6" lg="3">
+            <v-row no-gutters class="mr-2 my-2">
+              <v-col cols="12" sm="6" xl="5" class="pr-2">
+                <AppLabel>Overnight Care:</AppLabel>
+              </v-col>
+              <v-col cols="12" sm="6" xl="7">{{ licence?.overnightCare ? 'Yes' : 'No' }}</v-col>
+            </v-row>
+          </v-col>
+          <v-col cols="12" md="6" lg="4" xl="3">
+            <v-row no-gutters class="mr-2 my-2">
+              <v-col cols="12" sm="6" class="pr-2">
+                <AppLabel>Enrolled Spaces:</AppLabel>
+              </v-col>
+              <v-col cols="12" sm="6">{{ licence?.enrolledSpaces }}</v-col>
+            </v-row>
+          </v-col>
+          <v-col cols="12" md="6" lg="5" xl="6">
+            <v-row no-gutters class="mr-2 my-2">
+              <v-col cols="12" sm="6" xl="4" class="pr-2">
+                <AppLabel>Hours:</AppLabel>
+              </v-col>
+              <v-col cols="12" sm="6" xl="8">{{ licence?.operationFromTime }} - {{ licence?.operationToTime }}</v-col>
+            </v-row>
+          </v-col>
+        </v-row>
+      </v-expansion-panel-text>
+    </v-expansion-panel>
+  </v-expansion-panels>
+</template>
+
+<script>
+import AppLabel from '@/components/ui/AppLabel.vue'
+import { mapState } from 'pinia'
+import { useAppStore } from '@/stores/app'
+
+export default {
+  components: { AppLabel },
+  props: {
+    licenceDetails: {
```

Comment:
I would just call this licences to differentiate it from the component name. I.e. **LicenceDetails** displays a list of **licences**.

Reply:
as discussed, I will leave the way it is since we have Licence Details entity in CRM